components/breadcrumbs/breadcrumbs.js

```diff
@@ -0,0 +1,116 @@
+import './breadcrumb.js';
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { RtlMixin } from '../../mixins/rtl-mixin.js';
+
+class BreadCrumbs extends RtlMixin(LitElement) {
+	static get properties() {
+		return {
+			compact: {
+				type: Boolean,
+				reflect: true
+			}
+		};
+	}
+
+	static get styles() {
+		return css`
+			:host {
+				--d2l-breadcrumbs-background-color: rgb(251, 252, 252);
+				display: block;
+				position: relative;
+			}
+
+			:host::after {
+				background: -webkit-gradient(linear, left top, right top, from(rgba(255, 255, 255, 0)), to(var(--d2l-breadcrumbs-background-color)));
+				background: -webkit-linear-gradient(left, rgba(255, 255, 255, 0), var(--d2l-breadcrumbs-background-color));
+				background: -o-linear-gradient(left, rgba(255, 255, 255, 0), var(--d2l-breadcrumbs-background-color));
+				background: linear-gradient(to right, rgba(255, 255, 255, 0), var(--d2l-breadcrumbs-background-color));
```

Comment:
You can remove the vendor prefixes for `linear-gradient`. All the browsers we care about support the standard one:

IE v10, released September 2012
Firefox v16, released August 2012
Chrome v26, released February 2013
Safari v6.1, released October 2013
Opera 12.1, released November 2012

Reply:
Done ✔️